libraries/cyclestreets-view/src/main/java/net/cyclestreets/util/MapPack.kt

```diff
@@ -0,0 +1,63 @@
+package net.cyclestreets.util
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import net.cyclestreets.api.Maps
+import net.cyclestreets.api.Map
+import java.io.File
+import java.io.FileInputStream
+import java.io.FilenameFilter
+import java.io.IOException
+import java.util.*
+
+class MapPack private constructor(
+        private val map: Map
+) {
```

Comment:
I think this would read easier as a one-liner rather than these 3 lines, but maybe that's just me.